nccs/pipeline/direct/direct.py

```diff
@@ -135,60 +143,81 @@ def get_sector_exposure(sector, country):
     #     exp.gdf['value'] = exp.gdf.value
     #     exp.check()
 
-    if sector == 'agriculture':
-        exp = agriculture.get_exposure(crop_type="whe", scenario="histsoc", irr="firr")
+    # In this case a sub sector of agriculture is selected, this is only applied during the direct impacts
+    if sector.startswith('agriculture_'):
+        _, crop_type = agriculture.split_agriculture_sector(sector)
+        exp = agriculture.get_exposure(crop_type=crop_type, scenario="histsoc", irr="firr")
+
+    if sector == "agriculture":
+        # For agriculture, we need to combine the exposures for the different crop types. Since we
+        # have already merged the yearsets for the different crop types.
+        exps = [agriculture.get_exposure(crop_type=crop_type, scenario="histsoc", irr="firr")
+                for crop_type in ["whe", "mai", "soy", "ric"]]
+        # Sum up the exposures
+        exp = exps[0].copy()
+        columns_to_sum = ["impf_RC",  "impf_",  "value"]
```

Comment:
We only want to sum 'value' here: the others are the IDs of impact functions, which don't change

Reply:
Aha, thanks!